tests/test_declaration.py

```diff
@@ -443,6 +447,111 @@ def test_directional_property(StyleClass):
     )
 
 
+@pytest.mark.parametrize(
+    "value, expected",
+    [
+        ([VALUE1], [VALUE1]),
+        ([VALUE1, VALUE3], [VALUE1, VALUE3]),
+        ([VALUE2, VALUE1], [VALUE2, VALUE1]),
+        ([VALUE2, VALUE3, 1, 2, VALUE1], [VALUE2, VALUE3, 1, 2, VALUE1]),
+        # Duplicates are kept, but "normalized" via validation.
+        (
+            [VALUE3, 1, VALUE3, "1", True, " 1", VALUE2],
+            [VALUE3, 1, VALUE3, 1, 1, 1, VALUE2],
+        ),
+        # Other sequences should work too.
```

Comment:
Might be worth adding some "more exotic" sequences here, like an iterator and a dict, just to prove it's not just working based on indexing.

Reply:
Those aren't sequences... Do we want to support iterators and mappings as well? How would we even handle mappings — take the values I guess? Iterating over a dict takes the keys by default, but that seems unlikely to be what anyone would ever want in this case.

Reply:
Sorry - this is me overthinking things. I had in my head that the validation logic would unpack any iterable - which the value validation will do - but it won't even get to that point because of the initial type check.

I don't have a good use case for using iterables or mappings, so feel free to ignore this one.